ark/phenotyping/cell_consensus_cluster.R

```diff
@@ -55,3 +65,10 @@ print("Writing consensus clustering")
 cellClusterData <- arrow::read_feather(cellClusterPath)
 cellClusterData$hCluster_cap <- hClust[as.character(cellClusterData$cluster)]
 arrow::write_feather(as.data.table(cellClusterData), cellConsensusPath)
+
+# save the mapping from cluster to hCluster_cap
+print("Writing SOM to meta cluster mapping table")
+hClustLabeled <- as.data.table(hClust)
+hClustLabeled$cluster <- as.integer(rownames(hClustLabeled))
+hClustLabeles <- setnames(hClustLabeled, "hClust", "hCluster_cap")
```

Comment:
I think column names should be "cluster" and "hCluster_cap"

Reply:
Same as above.